dashboard/app/controllers/robots_controller.rb

```diff
@@ -0,0 +1,27 @@
+class RobotsController < ApplicationController
+  DISALLOWED_ROUTES = [
+    "/admin/",
+    "/api/",
+    "/blockly/",
+    "/dashboardapi/",
+    "/join/",
+    "/milestone/",
+    "/projects/",
+    "/sections/"
+  ].to_set.freeze
+
+  def index
```

Comment:
I'd recommend we also add some cache control headers, probably 1 day public? This will ensure that Cloudfront returns the robots.txt and avoid load on our servers. In the unit test we can then assert the appropriate cache control header is present.

```
expires_in 1.day, public: true
```

Reply:
Do we not bust the cloudfront cache on DTPs?

Reply:
Do you have an example of adding cache control headers I can go off of?

Reply:
In our case, I believe an explicit cache header is unnecessary; [we configure Cloudfront to cache everything by default](https://github.com/code-dot-org/code-dot-org/blob/d28ce11ba6ccc8956336d1d4e7617f697fccc124/lib/cdo/http_cache.rb#L347-L351)

Reply:
> Do we not bust the cloudfront cache on DTPs?

Yes, correct! We do indeed

Reply:
I think so, but I also think it's worth confirming manually after we deploy